src/store/index.ts

```diff
@@ -7,21 +7,22 @@ import router from '@/router';
 import Swal from 'sweetalert2';
 // types
 import { AppState } from '@/types/AppTypes.interface';
-import { User, Event, Feedback } from '@/types/FirebaseTypes.interface';
+import { User, Event, Feedback, Project } from '@/types/FirebaseTypes.interface';
 
 const getInitState = (): AppState => {
     return {
         user: auth.currentUser,               // firebase auth user
         isSideNavCollapsed: true,             // bool to check if sidenav is showing
         isLoading: true,                      // bool to keep track whether user is being retreived from the DB
-        is_under_maintenance: true,           // bool to know whether website is under maintenance and display maintenance screen
+        is_under_maintenance: false,          // bool to know whether website is under maintenance and display maintenance screen
```

Comment:
Just asking, do we need `is_under_maintenance` turn to false in staging ?

Reply:
no idts, was curious why it's set in staging as well

Reply:
Okay, i think we can leave it as false